src/components/AdminV2/cards/NumberCard/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Icon, Card, Collapsible,
+} from '@openedx/paragon';
+
+import DetailsAction from './DetailsAction';
+
+const NumberCard = ({
+  title, icon, description, detailActions,
+}) => {
+  const handleDetailsActionClick = (event) => {
+    const element = document.getElementById('learner-progress-report');
```

Comment:
Any thoughts on using `userRef` in place of `document.getElementById`?

Reply:
I did a bit of search on this, here are my findings. We can use ref here but we need to create a ref and pass it to both components i.e. `NumberCard` here and `LearnerReport` component. We can use Context but that may cause issues because refs are mutable and context should have immutable objects similar to state. 
The other solution is to pass it down from the common parent component but that would unnecessarily complicate the code. 

So, I think using `document.getElementById('learner-progress-report');` is the best approach for now.

PS: I copied the code fro the existing component over here.